package.json

```diff
@@ -111,6 +112,7 @@
     "check-hash": "1.0.1",
     "color": "1.0.3",
     "commander": "2.8.1",
+    "compare-versions": "^3.1.0",
```

Comment:
This shouldn't be a wildcard version, but a fixed version number.

Reply:
should `strip-ansi` also be fixed?